manual-tests.md

```diff
@@ -247,3 +247,14 @@
 
     * Check that it should end ok
     * Check in the Web that the icon is now what the YAML says
+
+
+# Test creating a macaroon with a specific expiration
+
+1. Take note of the current date. Pick a date two days later, and convert it
+   into ISO 8601. For example, midnight on January 1st, 2019 is
+   '2019-01-01T00:00:00'.
```

Comment:
How about `midnight on February 5th, 2019 is 2019-02-05T00:00:00` to also demonstrate that the month comes before the day? The order is reversed in many traditional formats.

Reply:
Indeed, that's probably more helpful. Done.